dockerhub/teamcity-server/README.md

```diff
@@ -44,7 +48,35 @@ To achieve that, you can pass an additional
 started with an alternative `server-https-proxy.xml` configuration file which enables HTTPS options.
 
 Alternatively, you can use a custom Tomcat configuration (see below).
-                       
+
+
+#### Configuring HTTPS Access to TeamCity Server
+
+If a TeamCity Server uses the [HTTPS connection](https://www.jetbrains.com/help/teamcity/using-https-to-access-teamcity-server.html), it transmits encrypted traffic through port `443` by default. 
+
+For security reasons, some operating systems impose restrictions
+on binding "privileged" ports (typically, ports below 1024) for non-root users, such as user 1000. As a result, port `443` can be unavailable for TeamCity Containers launched under _user 1000_.
+
+To avoid this issue, do one of the following:
+
+1. **(recommended) Map a non-privileged port on a host machine to the default HTTPS port inside the container**. This solution allows TeamCity to be accessible via HTTPS without running the server under the root user (which is otherwise required for accessing the privileged port `443`).
```

Comment:
Vise-versa

Reply:
Non-privileged port inside the container to the default port on a host

Reply:
fixed

Reply:
> Map the default HTTPS port inside the container to a non-privileged port on a host machine

Inside container we use custom 8443 port and map it to the default 443 port on the host. 
The docker client on the host requires ROOT privilege (it's why it can listen 443 port), and TeamCity server under user 1000 can use the custom 8443 port. 

Please, ping me if you have further questions

Reply:
right, I understand how it works. Didn't sleep today, sorry for sloppiness

> Map a non-privileged `8443` port inside the container to the default HTTPS port `443` on a host machine